### ISSUE TITLE
Problem install.

### ISSUE THREAD

**Issue by MirlynDevil** [ovflowd/chocolatey#361]
Hello,

I have this error : 

Warning. Please read Chocolatey Installation Tutorial. You need configure your servers DocumentRoot to the public folder of this CMS. Thanks!

But i have doing true configuration in .env  What is the problem ? My document root is public_html in my website i have do the Chocolatey CMS here.

**Comment by MirlynDevil** [comment]
[https://img4.hostingpics.net/pics/356947problem.png](url)

**Comment by TheOmanNorge** [comment]
Are you using cPanel? 
If not this might not be useful for you! 👎 

I've just installed Chocolatey myself the latest version, less then 1 hour ago. No joke. 
Got some errors myself, but easy enough to fix. 

But then again I'm Root admininstrator and also have access to WHM.
This might be a must, if you want to change the DocumentRoot of your domain.

This is how I did it; 

* Open up FileZilla, and login as root.
* Go to your user/domain that has Chocolatey installed on it. 
/var/cpanel/userdata/USERNAME/DOMAIN.COM

* Edit your domain. You will find DocumentRoot there and you can change it. 

> This is what I have:
>  documentroot: /home/chocolateycms/public_html/public

When done editing, save & Restart HTTP (Apache) service.

I also did this;

* Server configuration - Tweak Settings - Domains. 
* Restrict document roots to public_html = OFF.  This should NOT be a must as far as I know. 
That's mostly for Addon and Sub Domain, but did it anyway. 


Got more errors & more steps, but not gonna write the bible in here if not necessary ;)
As this might not be a problem for you, and was steps I took during the installation and so on.

I'm no expert at this, but please say if you encounter any problems!
I might be able to help.

Hope you get it working! 👍

**Comment by ovflowd** [comment]
Great support @TheOmanNorge next month great updates comming in chocolatey.

Also about the bugs can you report it?

**Comment by MirlynDevil** [comment]
I have cPanel but i can't change document root because i'm not administrator look but you can change documentroot in the CMS please for default ?

[https://img4.hostingpics.net/pics/949185icant.png] Look please i can't change /home/habbowo3 and me i would like to use for default this document root /home/habbowo3/public_html

**Comment by MirlynDevil** [comment]
Sant0ro, Have you solution for this problem ?

**Comment by TheOmanNorge** [comment]
Hi again! 😅 

Do you have subdomains, if you are about to say yes! 
Then this is also something you can do, tested it now and this also works! 👍 

* cPanel Control Panel - Subdomains

You can do it like i did, but that is perfectly up to you! 🚬 

![subdomains](https://user-images.githubusercontent.com/15328445/32903740-16b64d22-caf6-11e7-9b54-c6097d14e12f.png)

![subdomain](https://user-images.githubusercontent.com/15328445/32902999-2e1c59ea-caf4-11e7-956e-e40e4dd0c6dc.png)

![url](https://user-images.githubusercontent.com/15328445/32903219-c30a4c7e-caf4-11e7-9d49-a4fef1a2d14f.png)

You can use Redirects if you want your users to go straight to your subdomain. 
![redir](https://user-images.githubusercontent.com/15328445/32903574-9beaf52a-caf5-11e7-9279-7da461f545ae.png)



I installed it the normal way, but if you have problems along the way, please let me know! 👍 

If you don't have Subdomain or don't want to do it this way I'm sorry. Try contacting your hosting provider and ask them what they can do about this.

**Comment by TheOmanNorge** [comment]
Sorry @sant0ro
It was simple mistakes I did!. No Chocolatey bugs, sorry If I confused you with my text.

Just to mention what I did wrong; 
- Wrong PHP version when installing Composer. I had PHP v7.0 and needed to have 7.1
- Composer; Could not find driver. Forgot to install driver "php-mysqlnd" on the selected php version.

And yes I'm looking forward to the great updates that is coming to Chocolatey very soon! 😺

**Comment by MirlynDevil** [comment]
I have contacting my hosting provider and he said (can't change documentroot because I Have directly a domain website (mydomain.fr) (i not have subdomain) Please you can say me where you have buy your subdomain please because i really would like to test this CMS in french.

**Comment by MirlynDevil** [comment]
I can do this i think ! I have find cpanel control subdomain and domain :) https://img4.hostingpics.net/pics/602730look.png

**Comment by MirlynDevil** [comment]
You can help me please https://img4.hostingpics.net/pics/879676helpme.png

**Comment by MirlynDevil** [comment]
And on FileZilla what i'm doing and when i place chocolatey CMS what folder ?

**Comment by TheOmanNorge** [comment]
Place the content of Chocolatey inside public_html as before. 

![filemanager](https://user-images.githubusercontent.com/15328445/32917013-217b6bd4-cb1e-11e7-8388-a55adc487812.png)

![redirect](https://user-images.githubusercontent.com/15328445/32917092-5a059a42-cb1e-11e7-835e-ce1a1510b6a7.png)

**Comment by MirlynDevil** [comment]
Look [https://img4.hostingpics.net/pics/324217andnow.png](url)

**Comment by MirlynDevil** [comment]
Okay and what files i configure please ?

**Comment by TheOmanNorge** [comment]
Is it possible you have a Skype or something? I can help you out there if that makes it easier. 

You need to scroll down to the domain and select it.

**Comment by MirlynDevil** [comment]
Yes okay, i have skype add me : mimoun45140

**Comment by TheOmanNorge** [comment]
I have added you..

**Comment by TheOmanNorge** [comment]
Since 20 hours ago, I've helped this guy out. 

- The webhosting provider he has is bad.  
(No way to install Composer.) (Didnt have access to SSH/Terminal) 

- Wrong PHP version to begin with. But changed it to 7.1 did not work. 
- The webpage was showing 500 internal server message. 
- No way to change documentroot in a ok way without making subdomain. 

I've helped him out with a better solution, and I think he is happy now. 😄 
He has to find a better host next time that has all the requirements it needs in order to run Chocolatey.
